serializer.rst

```diff
@@ -4,19 +4,23 @@
 How to Use the Serializer
 =========================
 
-Serializing and deserializing to and from objects and different formats (e.g.
-JSON or XML) is a very complex topic. Symfony comes with a
-:doc:`Serializer Component </components/serializer>`, which gives you some
-tools that you can leverage for your solution.
+Symfony provides a serializer to serialize/deserialize to and from objects and
+different formats (e.g. JSON or XML). Before using it, read the
+:doc:`Serializer component docs </components/serializer>` to get familiar with
+its philosophy and the normalizers and encoders terminology.
 
-In fact, before you start, get familiar with the serializer, normalizers
-and encoders by reading the :doc:`Serializer Component </components/serializer>`.
+.. _activating_the_serializer:
 
-Activating the Serializer
+Installing the Serializer
 -------------------------
 
-The ``serializer`` service is not available by default. To turn it on, activate
-it in your configuration:
+Before using the serializer, run this command to install it in your application:
+
+.. code-block:: terminal
+
+    $ composer require serializer
+
+Then, enable the serializer in the framework config:
```

Comment:
This is needed? I see that indeed it is! Shouldn't we fix this with a recipe? It won't block this PR, but we should look into that.

Reply:
Wait, no, I think this is wrong. I think it auto-enables when the serializer is present.